test/basic.test.js

```diff
@@ -205,3 +205,47 @@ test('promise long resolve', (t) => {
     t.notOk(err)
   })
 })
+
+test('wait plugin registration', (t) => {
```

Comment:
What about test to cover error propagations with async await ?

```js
app.use(async () => {})
```

Reply:
Take a look [here](https://github.com/mcollina/avvio/blob/74ac2125913658002a21169bae690f417e0cfe98/test/async-await.js#L145-L158) :D

Reply:
I didn't saw it :D